ide/org.codehaus.groovy.eclipse.codeassist/src/org/codehaus/groovy/eclipse/codeassist/processors/TypeCompletionProcessor.java

```diff
@@ -130,7 +130,7 @@ public void setResolverInformation(ModuleNode module, JDTResolver resolver) {
      * </ul>
      */
     protected boolean doTypeCompletion(ContentAssistContext context, String expression) {
-        if (expression.isEmpty()) {
+        if (expression != null && expression.isEmpty()) {
```

Comment:
I also fixed this previously for all consumers of expression: b25e79ddeb58c80af94629cdb9e7f5cad3cfe741